doc/users.rst

```diff
@@ -427,16 +454,20 @@ compatibility issues.
 Intent Override
 ~~~~~~~~~~~~~~~
 
-.. code-block:: json
+.. code-block:: python
```

Comment:
Why is this block changed to python?

Reply:
It isn't a full json snippet so the parser was failing, I switched it to python so that we could still get the syntax highlighting